huaweicloud/services/lts/resource_huaweicloud_lts_cce_access.go

```diff
@@ -0,0 +1,534 @@
+package lts
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	"github.com/hashicorp/go-multierror"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/jmespath/go-jmespath"
+
+	"github.com/chnsz/golangsdk"
+
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/common"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/config"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/utils"
+)
+
+// @API LTS DELETE /v3/{project_id}/lts/access-config
+// @API LTS POST /v3/{project_id}/lts/access-config
+// @API LTS PUT /v3/{project_id}/lts/access-config
+// @API LTS POST /v3/{project_id}/lts/access-config-list
+func ResourceCceAccessConfig() *schema.Resource {
+	return &schema.Resource{
+		CreateContext: resourceCceAccessConfigCreate,
+		UpdateContext: resourceCceAccessConfigUpdate,
+		ReadContext:   resourceCceAccessConfigRead,
+		DeleteContext: resourceHostAccessConfigDelete,
+		Importer: &schema.ResourceImporter{
+			StateContext: cceAccessConfigResourceImportState,
+		},
+
+		Schema: map[string]*schema.Schema{
+			"region": {
+				Type:     schema.TypeString,
+				Optional: true,
+				Computed: true,
+				ForceNew: true,
+			},
+			"name": {
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+				Description: `Specifies the name of the cce access. The name consists of 1 to 64 characters.
+ Only letters, digits, underscores (_), and periods (.) are allowed, and the period cannot be the first
+ or last character.`,
+			},
+			"log_group_id": {
+				Type:        schema.TypeString,
+				Required:    true,
+				ForceNew:    true,
+				Description: `Specifies the log group ID.`,
+			},
+			"log_stream_id": {
+				Type:        schema.TypeString,
+				Required:    true,
+				ForceNew:    true,
+				Description: `Specifies the log stream ID.`,
+			},
+			"access_config": {
+				Type:        schema.TypeList,
+				MaxItems:    1,
+				Required:    true,
+				Elem:        cceAccessConfigDeatilSchema("access_config.0"),
+				Description: `Specifies the configurations of cce access.`,
+			},
+			"cluster_id": {
+				Type:        schema.TypeString,
+				Required:    true,
+				ForceNew:    true,
+				Description: `Specifies the CCE cluster ID.`,
+			},
+			"host_group_ids": {
+				Type:        schema.TypeList,
+				Elem:        &schema.Schema{Type: schema.TypeString},
+				Optional:    true,
+				Computed:    true,
+				Description: `Specifies the log access host group ID list.`,
+			},
+			"tags": common.TagsSchema(),
+			"binary_collect": {
+				Type:        schema.TypeBool,
+				Optional:    true,
+				Computed:    true,
+				Description: `Specifies whether binary collection. Default false.`,
```

Comment:
The description content is not right, seems `Default false`:
should be `Defaults to false` or `Default is false`

Reply:
done